charts/nexus-iq/templates/persistentVolumeClaim.yaml

```diff
@@ -1,28 +1,30 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: {{ template "iqserver.fullname" . }}-data
   labels:
-    {{- include "iqserver.labels" . | nindent 4 }}
+{{ include "iqserver.labels" . | indent 4 }}
```

Comment:
Maybe this is obvious to helm devs, but what's the motivation behind the changed indentation levels and the `indent` vs `nindent`?

Reply:
I believe in the old way, the `{{-` trimmed the content back to be on the same line with `labels:`, so a "newline indent" (`nindent`) brought it back down to the following line. 

I think I'd still like the template language to be indented to try to match the levels in the yaml. I've seen helm examples that maintained indentation that way. Are you finding other standards that lead to unindenting all the template code?